share/shaders/VolumeCellBase.frag

```diff
@@ -551,34 +653,46 @@ void main(void)
         int i = 0;
         do {
             intersections = FindInitialCell(eye, dir, t0, initialCell, entranceFace, t1);
+            skipCellWhenSearchingForInit = initialCell;
             
             if (intersections > 0) {
                 vec4 color = Traverse(eye, dir, rayLightingNormal, tMin, tMax, t1, initialCell, entranceFace, t1);
                 BlendToBack(accum, color);
             }
             
             // Failsafe to prevent infinite recursion due to float precision error
-            if (i++ > 8 || t1-t0 <= EPSILON)
+            if (i++ > 8) {
                 break;
+            }
+            if (t1-t0 <= EPSILON) {
```

Comment:
Is t1 guaranteed to be greater than t0 (i.e. no need to take absolute value)? Will the comparison with EPSILON work regardless of the user coordinates (it's not clear what coordinate system t0, t1 are in)

Reply:
`t` represents distance along the view ray as in `R=o+t*d`. This is supposed to test `t1 <= t0` rather than `t1 == t0`.